tools/ds-identify

```diff
@@ -1565,6 +1565,22 @@ dscheck_VMware() {
     return "${DS_NOT_FOUND}"
 }
 
+dscheck_WSL() {
+    if [ ! "${DI_VIRT}" = "wsl" ]; then
+        return "${DS_NOT_FOUND}"
+    fi
+
+    # The datasource needs to find the cloud-config files in the Windows host
+    # filesystem, which is exposed as 9p mount points, one per disk drive (partition).
+    # If none is found, the datasource cannot proceed.
+    if [ "$(grep -c '9p .*drvfs' "${PATH_ROOT}/proc/mounts")" = 0 ]; then
```

Comment:
Long options please (`grep --count`), for easier reading.

Also, matching on `'9p .*drvfs'` is still fairly broad - is there anything that we can do to tighten up this match?

Reply:
@holmanb Busybox's grep (as used by Alpine) only handles short options...

Reply:
@dermotbradley oh it does?

@CarlosNihelton I retract that part of the request, the other part still stands